src/pybind/mgr/orchestrator_cli/module.py

```diff
@@ -182,29 +196,82 @@ def _list_services(self, cmd):
                     s.version,
                     s.rados_config_location))
 
-            return HandleCommandResult(stdout="\n".join(lines))
+            return HandleCommandResult(odata="\n".join(lines))
 
-    def _service_add(self, cmd):
+    def _create_osd(self, inbuf, cmd):
+        """Create one or more OSDs
+
+        :cmd : Arguments for the create osd
+        """
+        #Obtain/validate parameters for the operation
+        cmdline_error = ""
+        if "drive_group" in cmd.keys():
+            params = cmd["drive_group"]
```

Comment:
Please don't accept raw JSON on the command line. We already have `inbuf` for that.